en/docs/guides/Provisioning/outbound-provisioning-with-salesforce.md

```diff
@@ -0,0 +1,190 @@
+# Configure Outbound Provisioning with Salesforce
+
+The WSO2 Identity Server (WSO2 IS) has the ability to provision users
+into different domains like Salesforce, Google, Facebook, etc., using
+its [identity provisioning framework](../../get-started/provisioning-architecture).
+
+This topic provides instructions on how to configure Salesforce as the
+Identity Provider to provision users from WSO2 Identity Server. The
+service provider in this scenario is WSO2 Identity Server. When WSO2 IS
+is the service provider, it is configured as the resident Service
+Provider. Therefore, after completing this tutorial you can see the
+users you add using WSO2 Identity Server being created in Salesforce
+too.
+
+## Configure Salesforce
+
+1.  Sign up as a Salesforce developer.
+    1.  Fill out the relevant information found in the following URL:
+        [https://developer.salesforce.com/signup  
+        ](https://developer.salesforce.com/signup)
+    2.  Click **Sign me up**.
+    3.  Click **Allow** to enable Salesforce to access your basic
+    information. This message pops up only when you log in to Salesforce
+    for the first time.
+
+    !!! note    
+        This document is explained using the Salesforce lightning theme. If
+        you are using the classic theme, click **Switch to Lightning Experience** on the top panel. 
+
+        ![lighteninig-experience](../assets/img/guides/switch-to-lightening.png)
+
+        You will be navigated to the lightening theme of Salesforce.
+
+        ![welcome-to-lightening.png](../assets/img/guides/welcome-to-lightening.png) 
```

Comment:
WDYT about breaking some of this into shorter subsections in this guide?  For example, "Sign up as a Salesforce developer", "Add connected app", "Configure profile", etc..

Personally, I feel shorter sections of steps makes the guide more appealing to follow and neater. But that's just my view and it may not apply depending on the nature of the content. Please consider and see if it makes sense to you and to this content.

Reply:
The problem is that, we keep referring to the steps here and there. Having them in different sections, would make the user feel that he needs to refer to a whole new section. That's why wanted to have the steps in the same section. 

Have divided the WSO2 IS side configs using separators. Hope this makes it clearer?